internal/genny/build/_fixtures/coke/go.mod

```diff
@@ -0,0 +1,100 @@
+module coke
+
+go 1.17
```

Comment:
Would this work when building against Go 1.16 (We still run tests there).

Reply:
Yes, this is just a newly generated project with cli and it defaults to the installed go version (1.17 in my setup).

I will update that again when gobuffalo/buffalo is completely free of packed and then tackle the Todo that mentioned generating and building a new buffalo project.